tests/prepdata/test_supergraph_construct.py

```diff
@@ -0,0 +1,74 @@
+import torch
+
+from kale.prepdata.supergraph_construct import SuperEdge, SuperGraph, SuperVertex, SuperVertexParaSetting
+
+# create three supervertices
+node_feat = torch.randn(4, 20)
+edge_index = torch.tensor([[0, 1, 2, 3], [1, 2, 3, 4]])
+edge_type = torch.tensor([0, 0, 1, 1])
+edge_weight = torch.randn(4)
+
+
+def test_supervertex():
+    """Test the supervertex construction."""
+
+    supervertex1 = SuperVertex("1", node_feat, edge_index)
+    supervertex2 = SuperVertex("2", node_feat, edge_index, edge_type)
+    supervertex3 = SuperVertex("3", node_feat, edge_index, edge_type, edge_weight)
+
+    assert supervertex1.__repr__()
+    assert supervertex2.edge_weight is None
+    assert supervertex3.edge_weight is not None
+
+    supervertex1.add_in_supervertex("2")
+    assert supervertex1.in_supervertex_list == ["2"]
+
+    supervertex2.add_out_supervertex("3")
+    assert supervertex2.out_supervertex_list == ["3"]
+
+
+supervertex1 = SuperVertex("1", node_feat, edge_index)
+supervertex2 = SuperVertex("2", node_feat, edge_index, edge_type)
+supervertex3 = SuperVertex("3", node_feat, edge_index, edge_type)
+
+# determine the supervertices among them
+edge_index = torch.tensor([[0, 1, 2, 3], [1, 1, 3, 3]])
+
+
+def test_superedge():
```

Comment:
Same as the above [comments](https://github.com/pykale/pykale/pull/338/files#r949364836).